project.clj

```diff
@@ -7,7 +7,9 @@
   :dependencies [[me.raynes/fs "1.4.6" :exclusions [org.clojure/clojure]]
                  [me.raynes/conch "0.8.0"]
                  [clj-time "0.6.0"]
+                 [cheshire "5.7.0"]
                  [prismatic/schema "1.0.4"]
+                 [circleci/clj-yaml "0.5.5"]
```

Comment:
Yeah the more I think about JSON vs YAML, I'm in favor of JSON.

If we go that route we can obviously delete this dependency.